src/ext/java/org/opentripplanner/ext/flex/FlexibleTransitLeg.java

```diff
@@ -0,0 +1,252 @@
+package org.opentripplanner.ext.flex;
```

Comment:
Is it intended that this diff is the way it is? Should it not have been changed in #3896 ?

Reply:
Will rebasing make the diff a lot smaller?

Reply:
This was built on top of #3896, so the same commit is in both PRs. I merged in the merge commit and now the diff is prettier.

Reply:
Much better!